editoast/src/core/mod.rs

```diff
@@ -306,10 +282,10 @@ pub struct StandardCoreError {
     context: HashMap<String, Value>,
     message: String,
     #[serde(default = "CoreErrorCause::default")]
-    cause: CoreErrorCause,
+    pub(crate) cause: CoreErrorCause,
```

Comment:
```suggestion
    pub cause: CoreErrorCause,
```

since this becomes part of the public `crate core` interface (used in views), let's make all the fields `pub` for api consistency

Reply:
Done.